api-reference/openapi.yaml

```diff
@@ -220,6 +236,38 @@ components:
                         type: string
                         example: Ctrl Hub deliver best in class software for managing your compliance, safety, risk and resource requirements
                         description: A description for the organisation
+    organisationUpdate:
+      description: ""
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: object
+                properties:
+                  type:
+                    type: string
+                    example: organisations
+                    description: The type of resource
+                    enum:
+                      - organisations
+                  id:
+                    type: string
+                    example: 1
```

Comment:
I'd probably grab the ID example for Ctrl Hub from another ref and put it here - I've been trying to make sure that the ID's are consistent because they'll cross over a bit in the future with include/related

Reply:
👉🏼  https://github.com/ctrl-hub/doc.public/blob/9734e518e6b24a8f4089eb8a68029e947bc7aa64/api-reference/openapi.yaml#L285

Reply:
(it's also UUID format vs String)

Reply:
Done!

Reply:
Just the format to change

Reply:
(https://github.com/ctrl-hub/doc.public/blob/9734e518e6b24a8f4089eb8a68029e947bc7aa64/api-reference/openapi.yaml#L283)